frontend/src/sass/includes/rc-collapse.scss

```diff
@@ -0,0 +1,61 @@
+.rc-collapse {
+  background-color: #f7f7f7;
+  border-radius: 3px;
+  border: 1px solid #d9d9d9;
+}
+.rc-collapse-anim-active {
+  transition: height 0.2s ease-out;
+}
+.rc-collapse > .rc-collapse-item {

```

Comment:
You can be more DRY by using the nesting syntax provided by SCSS. 

That goes for all the other selectors in this file as well.